dynaboard.yml

```diff
@@ -727,26 +856,32 @@ nodes:
     type: TEXT
     name: textStepInteractingWithComponentsBody
     properties:
-      height: '3'
+      height: '5'
+      isEnabled: '{{true}}'
+      isVisible: '{{true}}'
       renderMode: markdown
+      showExternalIndicator: '{{false}}'
       styles: ''
       top: '2'
       value: >-
         Below we have a small user account management example that includes a
-        data table, text input, and dropdown. With `Test` mode selected, choose
```

Comment:
No good deed goes unpunished?

Reply:
This one is not good grammar. It should be 
> With `Test` mode selected choose an account type, search for an account owner, or sort the data table.

the comma after selected is out of place, but the other one makes sense

Reply:
"With Test mode selected" is a dependent clause, so it should be separated using a comma.

Reply:
There is an understood you beginning the following independent clause.